### PR TITLE
Fix linter warns and errors for test_runner.sh script

### PR DESCRIPTION
**Pull request by la10736**
Just fixed warns and errors reported by linter in this sccript

### CODE REVIEW

File: e2e-tests/test_runner.sh [hunk 1]
```diff
@@ -49,13 +51,20 @@ if [ ! -f bin/zombienet-linux-x64 ]; then
...
 then
-    echo -e "${TXT_BIGRN}INFO: ${TXT_BIBLK}Running tests with a debug build${TXT_NORML}"
-    BUILDSUBPATH="debug"
-else
     echo -e "${TXT_BIGRN}INFO: ${TXT_BIBLK}Running tests with a release build${TXT_NORML}"
-    BUILDSUBPATH="release"
+elif [[ ${BUILDSUBPATH} == "debug" ]];
+then
+    echo -e "${TXT_BIGRN}INFO: ${TXT_BIBLK}Running tests with a debug build${TXT_NORML}"
 fi
```
Comment (drgora) [inline e2e-tests/test_runner.sh:68]: It looks like we could replace lines 62-68 with a single one:

```suggestion
echo -e "${TXT_BIGRN}INFO: ${TXT_BIBLK}Running tests with a ${BUILDSUBPATH} build${TXT_NORML}"
```

**Review by drgora — APPROVED** [review]
LGTM, I just left a minor suggestion.

**Comment by la10736** [comment]
I stopped last CI run to avoid wasting CI time.

File: e2e-tests/test_runner.sh [hunk 2]
```diff
@@ -12,14 +12,16 @@
...
 # ANSI color handles
-TXT_BICYA="\033[96;1m"
-TXT_BIPRP="\033[95;1m"
 TXT_BIBLU="\033[94;1m"
 TXT_BIYLW="\033[93;1m"
 TXT_BIGRN="\033[92;1m"
 TXT_BIRED="\033[91;1m"
 TXT_BIBLK="\033[90;1m"
 TXT_NORML="\033[0m"
+# shellcheck disable=SC2034
```
Comment (JackPiri) [inline e2e-tests/test_runner.sh:21]: Why not removing those two colors at all since they are not used?

**Review by JackPiri — APPROVED** [review]
The PR is fine to me.
Thanks

**Pull request merged**